config/locales/server.en.yml

```diff
@@ -1686,7 +1686,7 @@ en:
     alert_admins_if_errors_per_minute: "Number of errors per minute in order to trigger an admin alert. A value of 0 disables this feature. NOTE: requires restart."
     alert_admins_if_errors_per_hour: "Number of errors per hour in order to trigger an admin alert. A value of 0 disables this feature. NOTE: requires restart."
 
-    categories_topics: "Number of topics to show in /categories page."
+    categories_topics: "Number of topics to show in /categories page. If set to 0, it will automatically try to find a value to keep the two columns symmetrical (categories and topics). At least 5 topics will always be returned."
```

Comment:
Not sure about the "at least 5 topics will always be returned" part. We should definitely have a minimum when it's "automatic", but if the user wants less topics, they should be able to set it